<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="COMx">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="SPEED">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>40</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Disconnect">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>110</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="File">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>93</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>93</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>File...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Load_to_STM">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load to STM32</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>392</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
